### DISCUSSION TITLE
Изменения в погоде

### DISCUSSION THREAD

**Discussion by Vasilyochek** [octothorp-team/public-dobrograd#903] [Предложения]
Предлагаю поменять настройки аддона на погоду, если это возможно. Просто очень частый туман или тьма, которые настолько плотные, что видимость снижается до 1-5 метров, очень сильно мешают как в обычной игре, так и во время перестрелок и езды, даже не смотря на то, что в машине видимость поднимается в несколько раз. Нужно поднять дальность видимости вне машины хотя-бы до 20 метров, а в машине до 40 метров. Так хотя-бы можно будет увидеть свое авто на парковке или собеседника.

**Comment by Wani4ka** [comment]
Отключено в octothorp-team/gmod-dobrograd@91cf8afd1fe5fc1154dc33500c81b9b21a707a72
